src/Skeletor/Packages/Package.php

```diff
@@ -67,8 +67,12 @@ public function setName(string $name)
     /**
      * @return string
      */
-    public function getVersion()
+    public function getVersion(bool $allowEmpty = true)
     {
+        if ($allowEmpty === false) {
+            return empty($this->version) === true ? 'latest' : $this->version;
```

Comment:
I guess this one is personal preference. I like it for readability, but you are right that it is not really required.